src/components/modals/modification/StretchScale.vue

```diff
@@ -0,0 +1,51 @@
+<script setup lang="ts">
+import type Scale from "@/scale";
+import { ref } from "vue";
+import Modal from "../../ModalDialog.vue";
+import { adjustedLine } from "./utils";
+const props = defineProps<{
+  scaleLines: string[];
+  scale: Scale;
+}>();
+const emit = defineEmits(["update:scaleLines", "cancel"]);
+const amount = ref(1.005);
+// TODO: Advanced options for precision
+function modify() {
+  const result: string[] = [];
+  for (let i = 0; i < props.scaleLines.length; ++i) {
+    const monzo = props.scale.getMonzo(i + 1);
```

Comment:
This fails if there are invalid lines. Need to come up with a more robust way of dealing with scale line format preferences.